Siww online (#140)
subnero1/subnero1.github.io: _posts/2020-11-19-Subnero-at-SIWW-online.md

---
layout: post
title: Subnero attends SIWW online
banner : images/banner-pulse-standard.jpg
date:  2020-11-19
categories: wnc
thumbnail: images/pulse-thumbnail-siww-online.jpg
---

> Due to the pandemic, Singapore International Water Week has been postponed to July 2021.

Reviewer: ```suggestion
Due to the pandemic, Singapore International Water Week has been postponed to July 2021. 
```

> Instead of waiting until then, the organisors arranged a virtual show.

Reviewer: ```suggestion
Instead of waiting until then, the organizers arranged a virtual show.
```

> Subnero had a virtual booth with personnel available on line, permanently, to attend any visitors' queries.

Reviewer: ```suggestion
Subnero had a virtual booth with personnel available online, permanently, to attend any visitors' queries.
```

> Subnero had a virtual booth with personnel available on line, permanently, to attend any visitors' queries.

Reviewer: Can we mention what was displayed in the booth and have a link to the SWAN White paper and the modem brochure?

Author: Good idea, I also need to add the link to the website to SIWW.
